crime-applications-adaptor/src/main/resources/schemas/crimeapplicationsadaptor/maat_application_internal.json

```diff
@@ -153,7 +160,8 @@
     "solicitorAdminEmail",
     "courtCustody",
     "dateCreated",
-    "passported"
+    "passported",
+    "assessment"
```

Comment:
As I was sort of alluding to in Slack, personally I think we should make this not required. It sounds like Crime Apply won't send an empty means assessment but instead not send the field. For me it's also better if we don't send an assessment with empty means and instead just don't send an assessment object. Happy to discuss this one though. :)